alacritty_terminal/src/term/mod.rs

```diff
@@ -2184,6 +2193,28 @@ impl<T: EventListener> Handler for Term<T> {
     }
 }
 
+/// Terminal version for escape sequence reports.
+///
+/// This reports Alacritty's current version as a unique number based on its semver version. The
+/// different versions are padded to ensure that a higher semver version will always report a
+/// higher version number.
+fn version_number() -> usize {
+    let mut version = env::var("CARGO_PKG_VERSION").unwrap_or_default();
```

Comment:
This is not a correct way to so. You have this variable if you run alacritty from `cargo, e.g. `cargo run`. However this variable is not presented when you use `alacritty` just by executable from your `PATH`. So the version should be defined during compile time and not runtime.